### PR TITLE
Defect DS-410 fixed

### PR DESCRIPTION
**Pull request by Utcrash**
@jugnu-appveen - the discard draft api was not working. Removing the /utils as present in the service and faas apis made it work.  Let me know if the api url needs any change or needs to be reverted back

### CODE REVIEW

File: src/app/home/b2b-flows/b2b-flows.component.ts [hunk 1]
```diff
@@ -182,12 +182,12 @@ export class B2bFlowsComponent implements OnInit, OnDestroy {
...
           if (flow.status === 'Draft') {
-            request = this.commonService.put(
+            request = this.commonService.delete(
               'partnerManager',
-              `/${this.commonService.app._id}/flow/utils/` + id
+              `/${this.commonService.app._id}/flow/` + id
             );
           } else {
-            request = this.commonService.put(
+            request = this.commonService.delete(
```
Comment (jugnu-appveen) [inline src/app/home/b2b-flows/b2b-flows.component.ts:190]: Revert this change, The above change is fine.

**Review by jugnu-appveen — CHANGES_REQUESTED** [review]

File: src/app/home/b2b-flows/b2b-flows.component.ts (hunk 1 above)
Comment (Utcrash) [reply src/app/home/b2b-flows/b2b-flows.component.ts]: Made the change

**Review by Utcrash — COMMENTED** [review]
Reverted it back

**Review by jugnu-appveen — APPROVED** [review]
